scripts/release-post.js

```diff
@@ -181,12 +181,21 @@ function renderPost (results) {
 function writeToFile (results) {
   const filepath = path.resolve(__dirname, '..', 'locale', 'en', 'blog', 'release', `v${results.version}.md`)
 
-  if (fs.existsSync(filepath)) {
-    return Promise.reject(new Error(`Release post for ${results.version} already exists!`))
-  }
+  return new Promise(function (resolve, reject) {
+    fs.access(filepath, fs.F_OK, function (err) {
+      if (!err) {
+        return reject(new Error(`Release post for ${results.version} already exists!`))
+      }
+
+      fs.writeFile(filepath, results.content, function (err1) {
```

Comment:
@lpinca Exactly the point. I thought people will not be confused when they read the code with `err1`.